<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>287</width>
    <height>224</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>System Monitor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>251</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="lblCPU">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>21</pointsize>
        </font>
       </property>
       <property name="text">
        <string>CPU: 0.0%</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblRAM">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>21</pointsize>
        </font>
       </property>
       <property name="text">
        <string>RAM: 0.0%</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblDisco">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Disco: 0.0%</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="lblVersao">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>180</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>v 1.5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSobre">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Sobre</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>287</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
